vars/utils.groovy

```diff
@@ -1,3 +1,11 @@
+def cosa_cmd(cosaDir, userBuilder, args) {
```

Comment:
How about `cosaCmd` to match the casing of other commands?

Also, maybe better to make this also use the `params = [:]` paradigm, that way users can omit `cosaDir` and `userBuilder` if the defaults are fine (i.e. `/srv/fcos` and no `sudo -u $user`). WDYT?